Library/Homebrew/cask/spec/cask/artifact/binary_spec.rb

```diff
@@ -20,7 +20,7 @@
     shutup do
       Hbc::Artifact::Binary.new(cask).install_phase
     end
-    expect(FileHelper.valid_alias?(expected_path)).to be true
+    expect(expected_path).to be_a_valid_symlink
```

Comment:
I think this is more idiomatic in this case, and you are still able to look it up by name as you would be for `FileHelper.valid_alias?`.